include/SDL3/SDL_events.h

```diff
@@ -845,6 +845,9 @@ typedef struct SDL_ClipboardEvent
     SDL_EventType type; /**< SDL_EVENT_CLIPBOARD_UPDATE */
     Uint32 reserved;
     Uint64 timestamp;   /**< In nanoseconds, populated using SDL_GetTicksNS() */
+    bool isOwner;       /**< are we owning the clipboard (internal update) */
+    const char **mime_types; /**< current mime types */
```

Comment:
```suggestion
    const char **mime_types; /**< current mime types, may be NULL */
```
What does NULL mime types mean? There's nothing on the clipboard? You have to query what's on the clipboard?

There's also a race condition here, you might get an event that there's text on the clipboard, but by the time you get it, it's been replaced with an image. It might be better to remove these entirely and add a query to see whether you own the clipboard so people can get the current state when it happens?

Reply:
That race condition always exists as by design on almost all platforms there's a "supported mime types" phase and a "retrieve corresponding data in that mime type" phase, and perhaps except on wayland, this is never an atomic op. Client apps must be ready to handle the case where text was announced but is not available anymore.

NULL was meant to be the value when it's an internal update (or not supported yet as with Mac and Android), but I can also do a copy of `SDL_VideoDevice->mime_types` so that the event is consistent for external and internal updates.

Reply:
Yes please, consistency would be great.